roles/serviceassurance/tasks/component_qdr.yml

```diff
@@ -9,10 +9,11 @@
         namespace: '{{ meta.namespace }}'
       spec:
         deploymentPlan:
-          size: 2
+          size: 1
           role: interior
           livenessPort: 8888
           placement: Any
+          resources: {}
```

Comment:
If this just to pass the scorecard? If you're going to define a resources block is should really define some requests, at least.

Reply:
I was having issues with the Operator looping against the creation of the `component_qdr.yaml` where a change was being applied, so the Operator would never stop updating the k8s resource, leading to the `generation` of the object increasing indefinitely (and the Operator never taking a break from running Ansible).

However, I think there might have been some other issues going on in the test environment that were due to the use of the quickstart script you provided. I haven't investigated yet to figure out why that would cause any sort of issue (maybe some sort of race condition, or just weird out of order stuff).

I'm going to pull this out along with a couple other things that I think should be unnecessary and see if the patching issue returns. If not, then I will pull this out of the PR. If so, then we'll need to leave it in.

(The QDR Operator adds this automatically when undefined. I was just trying to make the spec match fully.)

Reply:
I have resolved this now. The `resource: {}` was unnecessary and has been dropped.